src/nepiada/env/nepiada.py

```diff
@@ -370,19 +373,20 @@ def get_rewards(self):
         """
         rewards = {}
 
-        # Get the average distance of the agents from target
+        # Get the average vector distance of the agents from target
         target_x = self.config.size / 2
         target_y = self.config.size / 2
-        global_arrangement_reward = 0
+        global_arrangement_vector = np.array([0.0, 0.0])
 
         for agent_name in self.agents:
             agent = self.world.agents[agent_name]
-            distance = np.sqrt(
-                (agent.p_pos[0] - target_x) ** 2 + (agent.p_pos[1] - target_y) ** 2
-            )
-            global_arrangement_reward += distance
+            global_arrangement_vector += np.array([(agent.p_pos[0] - target_x), (target_y - agent.p_pos[1])])
```

Comment:
This is a very important change so I would like a second opinion here.

Reply:
Yup, I'll modify them in my next commit

Reply:
Addressed in [727d9f1](https://github.com/BoundlessDevelopment/Capstone-Project/pull/43/commits/727d9f1b47c9d98ff494467d11971e87aebdcdf6)